imdg-clients.txt

```diff
@@ -1,3 +1,16 @@
+========== Java Client Standalone
+---
+Version: 5.5.0-BETA
+Date: 10/14/2024
+Download_OS: https://repo1.maven.org/maven2/com/hazelcast/hazelcast-java-client/5.5.0-BETA/hazelcast-java-client-5.5.0-BETA.jar
+Download_OS_Size: 6.1 MB
+Download_EE: https://hazelcast.jfrog.io/artifactory/release/com/hazelcast/hazelcast-enterprise-java-client/5.5.0-BETA/hazelcast-enterprise-java-client-5.5.0-BETA.jar
```

Comment:
Use `https://repository.hazelcast.com/release/com/hazelcast/hazelcast-enterprise-java-client/5.5.0-BETA/hazelcast-enterprise-java-client-5.5.0-BETA.jar`

Otherwise it asks me to login or kind of goes to UI and then download. The above is what we use for HZ. Try it first locally

Reply:
agreed with this

Reply:
fixed at fb9b8e7c0b14d20d933842d95b77ad928eb4a1a1